### PR TITLE
Added pack template test

### PR DESCRIPTION
**Pull request by dcorriveau-omniscient**
Related to https://github.com/fsprojects/Paket/issues/1414#issuecomment-173169406
The test is creating a Nuget package and as you can see if you unzip it, it doesn't include the output assembly (my test isn't very accurate because it's checking the file size but you get the point).

### CODE REVIEW

File: integrationtests/scenarios/i001376-pack-template/before/packages/MySql.Data/MySql.Data.nuspec [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
```
Comment (forki) [inline integrationtests/scenarios/i001376-pack-template/before/packages/MySql.Data/MySql.Data.nuspec]: scenarios/i001376-pack-template/before/packages/ should be excluded, right?

Comment (dcorriveau-omniscient) [inline integrationtests/scenarios/i001376-pack-template/before/packages/MySql.Data/MySql.Data.nuspec]: Yea.

**Comment by forki** [comment]
can yuou please rebase / squash that. so that we loose the history of that packages thingy

**Comment by dcorriveau-omniscient** [comment]
I tried something, let me know if the history is better.

**Comment by forki** [comment]
nope ;-) as you can see here it's now 2 commits more

**Comment by forki** [comment]
https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

**Comment by forki** [comment]
I assume you forgot to force push.

If you pull again and push, then you merge the old stuff in

**Comment by dcorriveau-omniscient** [comment]
Sorry, my doppelganger took control of my account and kept adding commits. Fortunately I managed to fix things up :P

**Comment by forki** [comment]
ha! look much better now :+1:

**Comment by forki** [comment]
fixed in https://github.com/fsprojects/Paket/pull/1420
thanks for creating a repro case

**Pull request merged**